pkg/deploy-manager/config.go

```diff
@@ -36,6 +36,7 @@ func init() {
 	nbv1.SchemeBuilder.AddToScheme(scheme.Scheme)
 }
 
+// StorageClusterOptions - silly comment enforced by linter
```

Comment:
I'd like to see comments about what it is and what it can do. I'm already confused with this blank struct.

Reply:
sure - this was just to pass the tests and see if we hit any other failures after that ;-)

However, I am really irritated by that linter thing failing on missing comments. it often leads to meaningless and redundant comments. :frowning_man:

Reply:
See my previous response to that complaint, I still like having the comments for review purposes.

Reply:
This one is totally my fault. Will update along with other fixes soon.

IMO,
1. Linter must fail, that is the only way to get documentation.(Easy to enforce)
2. The description should tell more about it than the name. (Difficult to enforce)

Reply:
I am all for documentation/comments - where reasonable.
I am against silly and redundant comments, which is what happens frequently...

Anyway: Just remember to run `make ocs-operator-ci` before creating a PR 😉

Reply:
`make`,`make build-functest` and `make ocs-operator-ci` all three passed :)